### PR TITLE
Jennifer/scan updates

### PR DESCRIPTION
**Pull request by Jennievon**
### Why this change is needed

Please provide a description and a link to the underlying ticket

https://github.com/ten-protocol/ten-internal/issues/2559
https://github.com/ten-protocol/ten-internal/issues/2560
https://github.com/ten-protocol/ten-internal/issues/2561
https://github.com/ten-protocol/ten-internal/issues/2562

### What changes were made as part of this PR

Please provide a high level list of the changes made

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/ten-protocol/ten-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

File: tools/obscuroscan_v3/frontend/src/components/modules/dashboard/recent-blocks.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export function RecentBlocks({ blocks }: { blocks: any }) {
           </Avatar>
           <div className="ml-4 space-y-1">
             <p className="text-sm font-medium leading-none">
-              #{block?.blockHeader?.number}
+              #{+block?.blockHeader?.number}
```
Comment (BedrockSquirrel) [inline tools/obscuroscan_v3/frontend/src/components/modules/dashboard/recent-blocks.tsx:16]: Is that `+` a typo or some javascript magic?

File: tools/obscuroscan_v3/frontend/src/lib/constants.ts [hunk 2]
```diff
@@ -2,12 +2,31 @@ export const socialLinks = {
...
 export const maxRetries = 3;
 export const pricePollingInterval = 60 * 1000;
 
+export const getOptions = (query: {
+  page?: string | string[];
+  size?: string | string[];
+}) => {
+  const options = {
+    offset: Number.isNaN(parseInt(query.page as string, 10))
+      ? 1
```
Comment (BedrockSquirrel) [inline tools/obscuroscan_v3/frontend/src/lib/constants.ts:18]: This should be zero by default, it's causing you to skip the first row.

File: tools/obscuroscan_v3/frontend/src/lib/constants.ts [hunk 3]
```diff
@@ -2,12 +2,31 @@ export const socialLinks = {
...
+      : parseInt(query.page as string, 10),
```
Comment (BedrockSquirrel) [inline tools/obscuroscan_v3/frontend/src/lib/constants.ts:19]: And this offset needs to be multiplied by the page size. I know that's a super weird API we should get that fixed but that seems to be how it is atm. So if you're on page 3 and your page size is ten you would send:

offset: 20, size: 10 -> that will give you rows 20-30

**Review by BedrockSquirrel — CHANGES_REQUESTED** [review]
LGTM except a bit of confusion on the pagination API

File: tools/obscuroscan_v3/frontend/src/components/modules/dashboard/recent-blocks.tsx (hunk 1 above)
Comment (Jennievon) [reply tools/obscuroscan_v3/frontend/src/components/modules/dashboard/recent-blocks.tsx:16]: It's same as using `{Number(block?.blockHeader?.number)}`

**Pull request merged**